lib/Sequence.fz

```diff
@@ -902,6 +899,18 @@ public Sequence(public T type) ref is
   public type.AS_STRING_NON_FINITE_MAX_ELEMENTS => 10
 
 
+  # equality
+  #
+  public fixed redef type.equality(a, b Sequence T) bool
```

Comment:
Does this work without `fixed`? I think it should, which would give a default implementation for `array`, `list`, etc.

Reply:
```
./build/lib/Sequence.fz:911:16: error 1: Failed to infer actual type parameters
        && ((a.zip b x,y->x=y) ∀ x->x)

In call to 'Sequence.zip', no actual type parameters are given and inference of the type parameters failed.
Expected type parameters: 'U, V'
Type inference failed for 2 type parameters 'U', 'V'

one error.
make: *** [Makefile:619: build/modules/base.fum] Error 1
```